content/docs/practices/remote_write.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Remote Write Tuning
```

Comment:
We've standardized on not using title case for menu items, so:

"Remote Write Tuning" -> "Remote write tuning"